package.json

```diff
@@ -49,11 +49,10 @@
       "prettier/prettier": [
         "error",
         {
-          "singleQuote": true,
```

Comment:
I believe all these settings can be removed, as they should be inferred from `.prettierrc` if not explicitly overridden.